jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -191,12 +191,13 @@ def step030_customize_settings(script_dir, puppet_settings) {
     }
 }
 
-def step035_customize_hocon_settings(script_dir, settings) {
+def step035_customize_hocon_settings(script_dir, settings, server_era) {
     if (settings == null) {
         echo "Skipping hocon settings customization; no overrides found in `hocon_settings`."
     } else {
         settings_json = JsonOutput.toJson(settings)
-        withEnv(["PUPPET_GATLING_HOCON_SETTINGS=${settings_json}"]) {
+        withEnv(["PUPPET_GATLING_HOCON_SETTINGS=${settings_json}",
+                 "PUPPET_SERVER_SERVICE_NAME=${server_era["service_name"]}"]) {
```

Comment:
As far as I can tell that's needed for the restart step inside of the customize_hocon_setings step